omega/omega-transaction/src/main/java/org/apache/servicecomb/saga/omega/transaction/TransactionAspect.java

```diff
@@ -57,7 +59,13 @@ Object advise(ProceedingJoinPoint joinPoint, Compensable compensable) throws Thr
     context.newLocalTxId();
 
     TimeAwareInterceptor interceptor = new TimeAwareInterceptor(this.interceptor);
-    interceptor.preIntercept(localTxId, signature, joinPoint.getArgs());
+    AlphaResponse response = interceptor.preIntercept(localTxId, signature, joinPoint.getArgs());
+    if (response.aborted()) {
+      String abortedLocalTxId = context.localTxId();
+      context.setLocalTxId(localTxId);
+      throw new InvalidTransactionException("Abort local sub transaction " + abortedLocalTxId +
```

Comment:
local transaction is sub transaction, "Aborted sub transaction... because global transaction ..."